gating/update_dependencies/run

```diff
@@ -67,6 +67,12 @@ for the dependencies to be updated regularly though, so that part continues
 to be done."
 fi
 
+
+# Execute role requirements file update
+export ROLE_REQUIREMENTS_FILE="${GATING_PATH}/../../ansible-role-${RPC_PRODUCT_RELEASE}-requirements.yml"
```

Comment:
I feel like the ordering here could be better, define the file then point to it, but it's a VERY minor nit.